app/src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
what its this line?

Reply:
that line allows to do unit test for Koltin instead of changing the class to "open", because the classes are final by default